src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -1,4 +1,8 @@
 package com.spartronics4915.frc2020;
 
 public final class Constants {
+    public static final int kFlywheelMasterID = -1;
```

Comment:
Convention is `kSubsystemDescriptorID`, just add `Launcher` right after the `k` for all